pkg/trait/logging.go

```diff
@@ -91,6 +91,7 @@ func (l loggingTrait) Apply(environment *Environment) error {
 			envvar.SetVal(&environment.EnvVars, envVarQuarkusLogConsoleJsonPrettyPrint, True)
 		}
 	} else if util.IsNilOrTrue(l.Color) {
```

Comment:
Is only when `l.Color` `nil` or `true` the right place?

Reply:
Oops. Good catch!

Reply:
Color defaults to true, which is why it would usually work ... but the logic should be independent here.